lib/order.rb

```diff
@@ -0,0 +1,90 @@
+require_relative 'customer'
+require "csv"
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+  #1,Lobster:17.18;Annatto seed:58.38;Camomile:83.21,25,complete
+  # def product_hash
+  #   product_string_array = []
+  #   CSV.read("data/orders.csv").map do |row|
+  #     product_string = row[1].to_s.split(";")
+  #     product_string_array << product_string
+  #   end
+  #   return product_string_array
+
+    # product_string_array.map do |string|
+    #   string.each do |i|
+    #     temp_string = i.split(":")
+    #     temp_array = []
+    #     temp_array << temp_string
+    #     temp_hash = temp_array.to_h
+    #     print temp_hash
+    #
+    #   end
+
+  def self.all
+    all_orders = []
+    product_string_array = []
+    # CSV.read("data/orders.csv").map do |row|
+    #   product_string = row[1].to_s.split(";")
+    #   product_string_array << product_string
+    #   temp_array = product_string_array.split(:)
+    #   temp_hash = temp_array.
+    # end
+    CSV.read("data/orders.csv").map do |row|
+      id = row[0].to_i
+      products = row[1]
+      customer = Customer.find(row[2].to_i)
+      fulfillment_status = row[3].to_s
```

Comment:
You wanted this to be a symbol, not a string:

```suggestion
      fulfillment_status = row[3].to_sym
```